lib/active_model/serializer/logging.rb

```diff
@@ -0,0 +1,32 @@
+module ActiveModel
+  class Serializer
+    module Logging
+      extend ActiveSupport::Concern
+
+      included do
+        cattr_accessor(:logger) do
+          ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT))
+        end
```

Comment:
So, there's a logger already defined on `ActiveModelSerializers.logger` which should be used.  it should default to the null logger and be configured in the railtie, which is extra work, I hope you don't mind

so, you could remove the line from the test helper since it defaults to null